### DISCUSSION TITLE
How to open my survey url inside the Phantom mobile app browser using the deeplink?

### DISCUSSION THREAD

**Discussion by samjoe5** [phantom/docs#214] [General]
In Magiceden when we click on connect wallet from the mobile it redirects to the Phantom mobile app browser. How I can do this for my survey URL to be opened in the Phantom mobile app browser

**Comment by adamdelphantom** [comment]
Hello @samjoe5,

For this you can use the [browse deep link.](https://docs.phantom.app/phantom-deeplinks/other-methods/browse)
